src/Common/src/System/Net/NTAuthentication.cs

```diff
@@ -3,6 +3,7 @@
 // See the LICENSE file in the project root for more information.
```

Comment:
I'm sure I'll be confused a few months from now seeing that we have two files with the same name in the repo.
Maybe naming this `NTAuthentication.Common.cs` would be disambiguating?